endpoint.py

```diff
@@ -0,0 +1,88 @@
+from fastapi import FastAPI, Depends, HTTPException, Query
```

Comment:
```suggestion
from fastapi import FastAPI, Depends, HTTPException, Query, status
```
エラーハンドルしているところにこちらを渡してあげると可読性が良くなりそうです！

```
...
get_task_or_exception(task_id=task_id,status_code=status.HTTP_404_NOT_FOUND, session=session)
...
```